packages/components/pages/f-registration/test/component/f-registration.component.spec.js

```diff
@@ -19,4 +19,21 @@ describe('Shared - f-registration component tests', () => {
         // Assert
         expect(registration.canBeClicked(link)).toBe(true);
     });
+
+    forEach([
+        ['termsAndConditionsLink', 'info/terms-and-conditions'],
+        ['privacyPolicyLink', 'info/privacy-policy'],
+        ['cookiesPolicyLink', 'info/cookies-policy']
+    ])
+    .it('should take you to the correct URL when clicking the `%s`', (linkName, expectedUrl) => {
+        // Arrange
+        const expectedUrlRegex = new RegExp(`^http(s?)://localhost:\\d+/${expectedUrl}$`);
```

Comment:
Could there be a trailing slash?

`^http(s?)://localhost:\\d+/${expectedUrl}\\/?$`

Reply:
Not at the moment. If one is added the test can be amended then